features/server_response_pvalue.feature

```diff
@@ -189,3 +189,18 @@ Feature: Server responses P value
     Then I invoke "sendNextSpan"
     And I wait to receive 1 trace
     Then a span name equals "span 3"
+
+
+  Scenario: Fixed sampling probability 1.0
+    Given I load scenario "FixedSamplingProbability1Scenario"
+    And I should receive no sampling request
+    Then I set the HTTP status code for the next requests to 200
+    And I invoke "sendNextSpan"
+    Then I should receive no traces
```

Comment:
This looks weird? Given a `samplingProbability` of `1.0` we should be receiving a trace with one span in it.